testsuite/python/elc.py

```diff
@@ -35,8 +35,8 @@ class ElcTest(ut.TestCase):
     def test_finite_potential_drop(self):
         s = self.system
 
-        p1 = s.part.add(pos=GAP + [0, 0, 1], q=+1)
-        p2 = s.part.add(pos=GAP + [0, 0, 9], q=-1)
+        p1 = s.part.add(pos=[0, 0, 1], q=+1)
```

Comment:
I thought it doesn't matter, where in the system the gap is? @reinaual 
Did this change influence the test outcome?

Reply:
This change did not alter the numerical result. I think the gap is always at the higher value of the z-axis:
https://github.com/espressomd/espresso/blob/127ad4ef7c9cee2b4242e76139cf69f55230d628/src/core/electrostatics_magnetostatics/elc.hpp#L92-L93
https://github.com/espressomd/espresso/blob/127ad4ef7c9cee2b4242e76139cf69f55230d628/src/core/electrostatics_magnetostatics/elc.cpp#L1196

If particles enter the gap, they are misclassified as particles from the dielectric contrast when calculating the forces and energies. The documentation warns against using ELC without walls to block particles from entering the gap:
https://github.com/espressomd/espresso/blob/127ad4ef7c9cee2b4242e76139cf69f55230d628/doc/sphinx/electrostatics.rst#L254-L259

Reply:
In theory it should not matter, to my knowledge in the implementation it does. The lower interface is assumed at `z=0` and the upper one at `z=h`, which is relevant for the error-estimation and the placement of the image charges.
To my understanding the testcase should be correct for every given prefactor, however setting it to `1e-100` allows to explicitly test the potential difference contribution but hides the misplaced particles in this case.